### ISSUE TITLE
Project documentation needs improving

### ISSUE THREAD

**Issue by jacksonj04** [GovLab/open-government-research-exchange#125]
After hunting down this repository out of interest in how it works behind the scenes and to look at possible improvements, the lack of a decent README.md or CONTRIBUTING.md makes it a pain to understand what the site is for, how it works and how you can make changes.

Might I suggest:

* [ ] Update the GitHub summary with a quick sentence, and the summary URL to reflect the production URL.
* [ ] Update README.md to include an overview of the site's purpose.
* [ ] Update README.md to include a 'quick start' guide to building the site, so developers can test what they do.
* [ ] If the spreadsheets in the README.md are intended to be public, make them so (they're currently restricted access on Google Drive).
* [ ] Add a CONTRIBUTING.md to explain the process for submitting/reviewing changes to the codebase.

**Comment by mocxd** [comment]
agree, this is something we ran out of time to do before launch, and should be addressed
